pkg/infra/pulumi_aws/plugin_iac.go

```diff
@@ -152,6 +154,50 @@ func (p Plugin) Transform(result *core.CompilationResult, deps *core.Dependencie
 	addFile("iac/k8s/add_ons/external_dns/index.ts")
 	addFile("iac/k8s/add_ons/index.ts")
 
+	addDir := func(dir string, exclusions ...string) {
+		var unreadEntries []string
+		dirContents, err := files.ReadDir(dir)
+
+		addEntries := func(parentDir string, entries []fs.DirEntry) {
+			for _, entry := range entries {
+				dirSuffix := ""
+				if entry.IsDir() {
+					dirSuffix = "/"
+				}
+				path := strings.TrimSuffix(parentDir, "/") + "/" + entry.Name() + dirSuffix
```

Comment:
nit: use `path.Join` instead of `+ "/" +` to be more platform agnostic

Reply:
The separator is always a forward slash when reading from `embed.FS`
From the docs for `go:embed`: 
> The //go:embed directive accepts multiple space-separated patterns for brevity, but it can also be repeated, to avoid very long lines when there are many patterns. The patterns are interpreted relative to the package directory containing the source file. **_The path separator is a forward slash, even on Windows systems._** Patterns may not contain ‘.’ or ‘..’ or empty path elements, nor may they begin or end with a slash. To match everything in the current directory, use ‘*’ instead of ‘.’. To allow for naming files with spaces in their names, patterns can be written as Go double-quoted or back-quoted string literals.
https://pkg.go.dev/embed#:~:text=The%20//go%3Aembed%20directive%20accepts,back%2Dquoted%20string%20literals.